src/include/cell.i

```diff
@@ -700,14 +715,15 @@ restart:
     unpack->start_txn = WT_TXN_NONE;
     unpack->stop_ts = WT_TS_MAX;
     unpack->stop_txn = WT_TXN_MAX;
+    unpack->oldest_durable_ts = WT_TS_NONE;
     unpack->newest_durable_ts = WT_TS_NONE;
```

Comment:
newest_durable_ts has to be initialized with WT_TS_MAX. All oldest members to be initialized with WT_TS_NONE and newest with WT_TS_MAX, indicating the validity window as (0, infinity).

Reply:
Fixed.